SofaKernel/modules/SofaTypes/src/sofa/defaulttype/Vec.h

```diff
@@ -0,0 +1,101 @@
+/******************************************************************************
```

Comment:
This compatibility file should stay in the original sofa/defaulttype ...so that there is no git mv.

Reply:
My POV was:
 - move the file with git mv (and keep history)
 - no more file in defaulttype then (defaulttype will become less messy overtime)
 - create the compatibility file (with the same path) inside Types ; when the deprecation time arrives, we just need to delete/disable the compat folder (and we dont forget to delete from Defaulttype, etc)

Reply:
If I remember correctly we decieded to have a systematic naming scheeme where namespace are exactly the same as the directory structure. 
So Vec.h it should be in somethine like: 
sofatypes::defaulttype
#include <SofaTypes/DefaultType/>
or
#include <sofatypes/defaulttypes/>       (I like this one most :))

And actually if find the name defaulttype not very informative. 
sofatypes/numerical/ would be a much better on informative. 
sofatypes/container/ etc...